linear_algebra/src/python-polynom-for-points.py

```diff
@@ -0,0 +1,139 @@
+from decimal import *
+getcontext()
+getcontext().prec=28
+
+count=0
+check=1
+zero_y=[]
+
+#number of points you want to use
```

Comment:
Our automated tests fail:
```
==================================== ERRORS ====================================

_______ ERROR collecting linear_algebra/src/python-polynom-for-points.py _______

linear_algebra/src/python-polynom-for-points.py:11: in <module>

    x=int(input("number of points: ").strip())

../../../virtualenv/python3.7.1/lib/python3.7/site-packages/_pytest/capture.py:661: in read

    raise IOError("reading from stdin while output is captured")

E   OSError: reading from stdin while output is captured

------------------------------- Captured stdout --------------------------------

number of points: 
```
so please indent the following lines underneath `if __name__ == "__main__":`

Reply:
I tried to improve the code! Thank you for your quick response.